core/store/migrate/migrations/0195_add_evm_chain_id_to_job_specs.go

```diff
@@ -0,0 +1,73 @@
+package migrations
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/pressly/goose/v3"
+
+	"github.com/smartcontractkit/chainlink/v2/core/services/chainlink"
+)
+
+func init() {
+	goose.AddMigrationContext(Up195, Down195)
+}
+
+const (
+	up195_1 = `
```

Comment:
Can we name these instead of numbering them?

Reply:
Sure